lib/asg-functions

```diff
@@ -169,4 +169,17 @@ asg-suspend() {
   done
 }
 
+asg-scaling-activities() {
+  # type: query
+  # show all scaling activities
+  local inputs=$(__bma_read_inputs $@)
+  local output=$(__bma_read_switches $inputs | grep ^--output | cut -d\  -f2-)
+  [[ -z "$inputs" ]] && __bma_usage "auto-scaling-group" && return 1
+  for asg in $(__bma_read_resources ${inputs}); do
```

Comment:
I'd suggest to follow some kind of a convention when we are working with variable names.  For example on line 176 there is "$inputs", but on line 178 there is "${inputs}".  I propose not ti use curly braces unless it is required (e.g. for expansions or when the variable is in the middle of a word).  Also, the code is very inconsistent with the quotation of variables, e.g. on line 177 "$inputs" are properly double quoted, but on other lines variables are passed unquoted.  It is quite important to follow strict quotation rules or we may get some unexpected surprises down the line. WDYT?